src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_params.py

```diff
@@ -9,6 +9,7 @@
 from azure.mgmt.resource.resources.models import DeploymentMode
 from azure.mgmt.resource.locks.models import LockLevel
 from azure.mgmt.resource.managedapplications.models import ApplicationLockLevel
+from azure.mgmt.resource.policy.models import PolicyMode
```

Comment:
move this into the `if` to avoid crashing in older api

Reply:
Ah, of course! thanks @yugangw-msft